src/xlsx/functions/cells.ts

```diff
@@ -33,11 +34,10 @@ export function addFormula(
     return { attrs: [], node: escapeXml`` };
   }
 
-  const attrs: XMLAttributes = [["t", type]];
+  const attrs: XMLAttributes = [["cm", "1"]];
   const XlsxFormula = adaptFormulaToExcel(formula);
-
   const exportedValue = adaptFormulaValueToExcel(value);
-  const node = escapeXml/*xml*/ `<f>${XlsxFormula}</f><v>${exportedValue}</v>`;
+  const node = escapeXml/*xml*/ `<f t="array" ref="${arrayFormulaRange}">${XlsxFormula}</f><v>${exportedValue}</v>`;
```

Comment:
also add a comment saying we are flagging all formulas as array because it works and it's easier (rather than detecting sub-expressions spilling)